cli/init.go

```diff
@@ -34,8 +55,9 @@ func initConfig(c *cli.Context) error {
 	if err != nil {
 		return fmt.Errorf("Failed to get plugin (%s), error: %s", pluginName, err)
 	}
+
```

Comment:
pls alos cahck if the cli has any other command, which uses a plugin and implement the same functionality

Reply:
I have tried them:

```
init           Init bitrise config.
  setup          Setup the current host. Install every required tool to run Workflows.
  version        Prints the version
  validate       Validates a specified bitrise config.
  update         Updates the Bitrise CLI.
  run            Runs a specified Workflow.
  trigger-check  Prints out which workflow will triggered by specified pattern.
  trigger        Triggers a specified Workflow.
  export         Export the bitrise configuration.
  normalize      Normalize the bitrise configuration.
  step-list      List of available steps.
  step-info      Provides information (step ID, last version, given version) about specified step.
  workflows      List of available workflows in config.
  share          Publish your step.
  plugin         Plugin handling.
  stepman        Runs a stepman command.
  envman         Runs an envman command.
  help           Shows a list of commands or help for one command
```

Only the init command has a problem if the plugins have not installed yet.